tests/functional/adapter/materialized_view_tests/test_materialized_views.py

```diff
@@ -163,19 +163,6 @@ def test_change_is_applied_via_alter_str_true(self, project, my_materialized_vie
         assert_message_in_logs(f"Applying ALTER to: {my_materialized_view}", logs)
         assert_message_in_logs(f"Applying REPLACE to: {my_materialized_view}", logs, False)
 
-    def test_change_is_applied_via_alter_str_false(self, project, my_materialized_view):
```

Comment:
The string version of this is already tested with the test above. Also, this does not represent a change, and just the fact that `"False"` is treated the same as `False`. This is covered in a unit test.